src/python/generate_singa_wrapper.sh

```diff
@@ -26,6 +26,8 @@ USR_LOCAL=/home/chonho/local
 
 swig -c++ -python -I../../include singa.i
 
+echo ${SRC_CC[@]}
+
 g++ -fPIC ${SRC_CC[@]} singa_wrap.cxx -shared -o _singa.so \
     -L${USR_LOCAL}/lib -lprotobuf -Wl,-rpath=${USR_LOCAL}/lib \
     -L../../lib -lsinga_core -lsinga_model -lsinga_utils -Wl,-rpath=../../lib \
```

Comment:
pls replace the hard-code paths, e.g., SINGA_ROOT and USR_LOCAL.
for -I<LIB_PATH>, the LIB_PATH should use the paths found by cmake.